beta.go

```diff
@@ -19,7 +17,7 @@ import (
 //
 // See http://mathworld.wolfram.com/BetaFunction.html for more detailed informations.
 func Beta(a, b float64) float64 {
-	return math.Exp(Lbeta(a, b))
+	return gonum.Beta(a, b)
```

Comment:
it would be interesting to check if the additinal indirection doesn't hurt performances too much.

(otherwise, a little of code duplication wouldn't be _that_ bad, I think)